code/controllers/subsystems/processing/ntsl2.dm

```diff
@@ -44,6 +45,8 @@ NTSL2 deamon management subsystem, responsible for handling events from deamon a
 /datum/controller/subsystem/processing/ntsl2/proc/handle_response(var/datum/http_response/response, var/command)
 	if (response.errored)
 		log_debug("NTSL2++: Proc error while performing command '[command]': [response.error]")
+		log_debug("NTSL2++: Due to proc error, NTSL2++ has been DISABLED.")
+		connected = FALSE
```

Comment:
Wouldnt it be cleaner to call `disconnect()` here instead of just setting connected to false?